package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -45,39 +45,74 @@
+for residues 5-10 of adenylate kinase (AdK). The trajectory is included within
+the test data files::
 
    import MDAnalysis as mda
    from MDAnalysisTests.datafiles import GRO, XTC
    u = mda.Universe(GRO, XTC)
-   r = u.select_atoms("protein")    # selection of residues
 
-   from MDAnalysis.analysis.dihedrals import Ramachandran
+   # selection of atomgroups
+   ags = [res.phi_selection() for res in u.residues[4:9]]
+
+   from MDAnalysis.analysis.dihedrals import Dihedral
+   R = Dihedral(ags).run()
+
+The angles can then be accessed with :attr:`Dihedral.angles`.
+
+Ramachandran analysis
+~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~MDAnalysis.analysis.dihedrals.Ramachandran` class allows for the
+quick calculation of phi and psi angles. Unlike the :class:`~MDanalysis.analysis.dihedrals.Dihedral`
+class which takes a list of `atomgroups`, this class only needs a list of
+residues or atoms from those residues. The previous example can repeated with::
+
+   u = mda.Universe(GRO, XTC)
+   r = u.select_atoms("resid 5-10")
+
    R = Ramachandran(r).run()
 
+Then it can be plotted using the built-in plotting method :meth:`Ramachandran.plot()`::
+
    import matplotlib.pyplot as plt
    fig, ax = plt.subplots(figsize=plt.figaspect(1))
-   ax.set_title("Ramachandran Plot (AdK)")
    R.plot(ax=ax, color='k', marker='s')
 
-Alternatively, if you wanted to plot the data yourself, the angles themselves
-can be accessed using :attr:`Ramachandran.angles`::
+The Janin class works in the same way, only needing a list of residues.
 
-   fig, ax = plt.subplots(figsize=plt.figaspect(1))
-   ax.axis([-180,180,-180,180])
-   ax.axhline(0, color='k', lw=1)
-   ax.axvline(0, color='k', lw=1)
-   ax.set(xticks=range(-180,181,60), yticks=range(-180,181,60),
-          xlabel=r"$\phi$ (deg)", ylabel=r"$\psi$ (deg)")
-   for ts in R.angles:
-       ax.scatter(ts[:,0], ts[:,1], color='k', marker='s')
+Reference plots can be added to the axes for both the Ramachandran and Janin
+classes using the kwarg ``ref=True``. These were made using data obtained from
+a large selection of pdb files, and were analyzed using these classes. The
+allowed and marginally allowed regions of the Ramachandran reference plt have
+cutoffs set to include 90% and 99% of the data points, and the Janin reference
+plot has cutoffs for 90% and 98% of the data points.
 
-Analysis Class
---------------
+These classes are prone to errors if the topology contains duplicate or missing
+atoms (e.g. atoms with `altloc` or incomplete residues). If the topology has as
+an `altloc` attribute, you must specify only one `altloc` for the atoms with
+more than one (``"protein and not altloc B"``).
+
+Analysis Classes
+----------------
+
+.. autoclass:: Dihedral
+   :members:
+   :inherited-members:
+
+   .. attribute:: angles
+
+       Contains the time steps of the angles for each atomgroup in the list as
+       an n_frames×len(atomgroups) :class:`numpy.ndarray` with content
```

Comment:
ooooh... proper typography!

Reply:
Put the variables in double-back ticks
```..rest
``n_frames×len(atomgroups)``
```
so that they get code formatting.

(Still ❤️ ing the ×)